data/moves.js

```diff
@@ -4863,10 +4863,10 @@ let BattleMovedex = {
 		priority: 3,
 		flags: {contact: 1, protect: 1, mirror: 1},
 		onTry: function (pokemon, target) {
-			if (pokemon.activeTurns > 1) {
+			if (pokemon.activeTurns > 1 || pokemon.lastMove) {
 				this.attrLastMove('[still]');
 				this.add('-fail', pokemon);
-				this.add('-hint', "Fake Out only works on your first turn out.");
+				this.add('-hint', "Fake Out only works if it's your first move, on your first turn out.");
```

Comment:
I would do it like this:

```suggestion
				this.add('-hint',
					pokemon.activeTurns > 1 ?
					"Fake Out only works on your first turn out." :
					"Fake Out only works if it's your first move, on your first turn out."
				);
```